Gemfile

```diff
@@ -52,6 +52,7 @@ group :development, :test do
   # Omakase Ruby styling [https://github.com/rails/rubocop-rails-omakase/]
   gem "rubocop-rails-omakase", require: false
   gem "erb_lint", require: false
+  gem "devpack"
```

Comment:
@bobf is this a good usage of [devpack](https://rubygems.org/gems/devpack)?

We try our best to keep the moneygun biolerplate LEAN and stick to defaults whenever we can.